config/initializers/rollbar.rb

```diff
@@ -5,9 +5,7 @@
   config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
 
   # Here we'll disable in 'test':
-  if Rails.env.test?
-    config.enabled = false
-  end
+  config.enabled = false if Rails.env.test?
```

Comment:
Many of that changes, the rubocop did himself, this change is one of them! I'll use your way! :)